packages/shared/src/components/profile/ProfileButton.tsx

```diff
@@ -42,7 +44,15 @@ export default function ProfileButton({
             )}
             onClick={wrapHandler(() => onUpdate(!isOpen))}
           >
-            <span className="ml-3 block">{user.reputation ?? 0}</span>
+            <ReputationUserBadge
+              className="ml-3 !typo-callout"
+              user={{
+                reputation: user.reputation ?? 0,
```

Comment:
In other usages we're sending the whole user object, why only reputation here? If only reputation is needed, can we send only that in all use cases?

Reply:
`ReputationUserBadge` needs only `reputation`. The `??` was from all code but since then I removed it 898401e701e81929b3a7df7bbbdb179025306491 because it is not really needed. So overall I think cleaner now.

Reply:
Not much to add but indeed we could have opted to have only `reputation` passed as we don't use anything else.
But either way works for me :)

Reply:
The good part if we have `reputation` as prop we can default it to 0 :)

Reply:
I wanted to keep it open you never know when they want to show different reputation icon based on role 😆